pkg/typesystem/typesystem.go

```diff
@@ -331,6 +331,54 @@ func (t *TypeSystem) DirectlyRelatedUsersets(objectType, relation string) ([]*op
 	return usersetRelationReferences, nil
 }
 
+// resolvesTypeRelationToDirectlyAssignable returns whether the assignable types ONLY if
+// the specified type and relation is exclusively assignable. Otherwise, it will return nil as slice.
```

Comment:
```suggestion
// resolvesTypeRelationToDirectlyAssignable returns whether the input object#relation is related ONLY to
// concrete types. 
// Otherwise, it will return nil as slice.
```